application/src/main/java/org/opentripplanner/street/model/edge/StreetTransitEntityLink.java

```diff
@@ -20,13 +23,13 @@ public abstract class StreetTransitEntityLink<T extends Vertex>
 
   static final int STEL_TRAVERSE_COST = 1;
 
-  private final T transitEntityVertex;
+  private final Vertex transitEntityVertex;
 
   private final Accessibility wheelchairAccessibility;
 
   protected StreetTransitEntityLink(
-    StreetVertex fromv,
-    T tov,
+    Vertex fromv,
+    TransitEntranceVertex tov,
     Accessibility wheelchairAccessibility
   ) {
```

Comment:
Would it be enough to have a constructor that accepts (`Vertex`, `Vertex`, `Accessibility`)?